src/uwtools/config.py

```diff
@@ -104,6 +123,25 @@ def dump_file(self, output_path):
         with open(output_path, 'w', encoding="utf-8") as file_name:
             yaml.dump(self.data, file_name, sort_keys=False)
 
+    def _yaml_include(self, loader, node):
+        ''' Returns a dictionary that includes the contents of the referenced
+        YAML files, and is used as a contructor method for PyYAML'''
+
+        filepaths = loader.construct_sequence(node)
+        cfg = {}
+        for filepath in filepaths:
+            abs_path = filepath
+            if not os.path.isabs(filepath):
+                abs_path = os.path.join(os.path.dirname(self.config_path), filepath)
```

Comment:
Wondering what the use of declaring the variable abs_path to hold the contents of filepath is here? Is this extra variable necessary to account for a side effect happening with the join and update on lines 135-136?

Reply:
That is definitely how I was using it here, yes. I kind of expected that the linter would complain about redefining a name from the outer scope. 

I ran a test just now and pylint is perfectly happy with updating `filepath` directly. I'll clean that up since it's a simpler approach. 

Thanks!